app-modules/audit/src/Filament/Resources/AuditResource/Pages/ViewAudit.php

```diff
@@ -55,9 +58,10 @@ public function infolist(Infolist $infolist): Infolist
                     ->schema([
                         TextEntry::make('auditable_type')
                             ->label('Auditable'),
-                        TextEntry::make('user.name')
+                        TextEntry::make('change_agent_name')
```

Comment:
Same as in the Edit page. `user.name` still needs to work and be present before the migrations finish and the feature is activated.